Doc/library/heapq.rst

```diff
@@ -82,6 +82,55 @@ The following functions are provided:
    on the heap.
 
 
+.. function:: heapify_max(x)
+
+   Transform list *x* into a heap, in-place, in linear time.
+
+   .. versionadded:: next
+
+
+.. function:: heappush_max(heap, item)
+
+   Push the value *item* onto the *heap*, maintaining the heap invariant.
+
+   .. versionadded:: next
+
+.. function:: heappop_max(heap)
+
+   Pop and return the largest item from the *heap*, maintaining the heap
+   invariant.  If the heap is empty, :exc:`IndexError` is raised.  To access the
+   largest item without popping it, use ``heap[0]``.
+
+   .. versionadded:: next
+
+
+.. function:: heappushpop_max(heap, item)
+
+   Push *item* on the heap, then pop and return the largest item from the
+   *heap*.  The combined action runs more efficiently than :func:`heappush_max`
+   followed by a separate call to :func:`heappop_max`.
+
+   .. versionadded:: next
+
+
+.. function:: heapreplace_max(heap, item)
```

Comment:
What's the differeence between this one and `heapreplace`. The descriptions are the same.